templates/core/terraform/api-webapp.tf

```diff
@@ -6,27 +6,21 @@ locals {
   version = replace(replace(replace(data.local_file.api_app_version.content, "__version__ = \"", ""), "\"", ""), "\n", "")
 }
 
-resource "azurerm_app_service_plan" "core" {
```

Comment:
I guess this causes the app service to be recreated, but assume that's not an issue as users won't have customised it?